LoopKitUI/Views/Settings Editors/TherapySetting+Settings.swift

```diff
@@ -14,15 +14,15 @@ extension TherapySetting {
     public func helpScreen() -> some View {
         switch self {
         case .glucoseTargetRange:
-            return AnyView(CorrectionRangeInformationView(onExit: nil, mode: .modal))
+            return AnyView(CorrectionRangeInformationView(onExit: nil, mode: .settings))
```

Comment:
I was surprised here...shouldn't these be `.acceptanceFlow` ?  Or, better yet, maybe `helpScreen()` should take a `mode: PresentationMode`?

Reply:
The help screen is always presented as a "card" (the ones in the acceptance flow have buttons and are part of the flow), which is why the `.settings` is hardcoded here (since that will have the appropriate buttons present)

Reply:
ahhh, ok.  a bit confusing but I get it now. 👍